### ISSUE TITLE
Tests with quotes in their names cannot be executed individually

### ISSUE THREAD

**Issue by kentcb** [Dart-Code/Dart-Code#2055]
Given this test:
```dart
void main() {
  test('a test with "quotes" in its name', () {});
}
```
Executing tests as a whole works fine:

![image](https://user-images.githubusercontent.com/1901832/66974302-03e11780-f0de-11e9-9014-c6d014fe119f.png)

But right-clicking and attempting to execute the test on its own results in the following debug console output:

```
No tests match regular expression "^a test with quotes in its name$".
Exited (1)
```

The quotes are stripped from the name and so there is no match.

**Comment by DanTup** [comment]
This might be covered by my fix for #2053 but I'll do some testing.

**Comment by DanTup** [comment]
Yep, seems like that fix covered this (and now has an automated test). Thanks for the report!

![Screenshot 2019-10-17 at 8 46 07 am](https://user-images.githubusercontent.com/1078012/66988531-98d21980-f0ba-11e9-9e85-f7fb44be6fa4.png)
